src/docs/framework-integration/vue.md

```diff
@@ -14,19 +14,43 @@ contributors:
 
 In order to use the custom element library within the Vue app, the application must be modified to define the custom elements and to inform the Vue compiler which elements to ignore during compilation. This can all be done within the `main.js` file. For example:
 
+### Add the component(s) to the app dependencies
+Typically done with `npm install --save test-components`
+
+
+### Import the components into the 'main.js' file
+by 
+
+- importing the node module
+- telling Vue to ignore the custom element tags
+- binding their code to the window object
+
+See `// HERE` comments:
+
```

Comment:
Just a neutral bystander reviewing here: if I were reading these docs I’d find the incomplete sentences jarring (especially if you’re trying to continue a sentence started in a heading that tries to continue into body text `… into the 'main.js' file … by`).

Further, the previous paragraph ends with “For example:”, and a new heading begins. Seems like the document flow is disrupted.

Instead, I think this entire section could be removed, and the previous sentence could be edited:

```diff
-In order to use the custom element library within the Vue app, the application must be modified to define the custom elements and to inform the Vue compiler which elements to ignore during compilation. This can all be done within the `main.js` file. For example:
+In order to use the custom element library within the Vue app, the application must be modified to define the custom elements and to inform the Vue compiler which elements to ignore during compilation. This can all be done within the `main.js` file. Assuming you’ve run `npm install --save test-components` beforehand, take the following  example:
```

Brevity is clarity!

Reply:
Agreed and thank you. 

I don't know how to get back into my suggested modifications to edit them.